src/style/pre-match.less

```diff
@@ -408,6 +416,11 @@ a.back:hover {
 			outline: 0;
 			color: white;
 		}
+
+		&.disabled {
```

Comment:
Disabled button state: 
<img width="602" alt="CleanShot 2021-12-02 at 17 21 57@2x" src="https://user-images.githubusercontent.com/199204/144376347-99e17bb5-f942-4f88-b4c2-0eac7a1831cc.png">

This wasn't visible in the MR demo video as I added it after taking the video.

Reply:
Good call re disabled button, I've pushed a grayscale version. This change also introduces a slight transition on the filter, let me know what you think:

![CleanShot 2021-12-03 at 15 09 07](https://user-images.githubusercontent.com/199204/144549145-66c30592-05a3-4e02-b174-46b535bb41ff.gif)

I didn't add browser prefixes for `transition` css property as they're no longer required in modern browsers.